init/postgres/init.sql

```diff
@@ -93,19 +93,1167 @@ insert into hr.departments (department_id, department_name) values (1,'Sales');
+insert into hr.salaries (salary_id, employee_id, effective_date, amount) values (2,2,'2015-07-19',216000.0);
+insert into hr.employees (employee_id, department_id, job_title, given_name,family_name, birth_date, hire_date) values (3,3,'Programmer','Avram','Lawrence','1994-Jan-16','2016-Mar-26');
+insert into hr.employee_address (address_id, employee_id, address_line1, address_line2, city, state, postal_code) values (3,3,'661-7101 Molestie. Av.','','Madison','88882','United States');
+insert into hr.employee_contacts (contact_id, employee_id, phone_number, email) values (3,3,'(551) 454-0653','Avram.Lawrence@acme.com');
+insert into hr.salaries (salary_id, employee_id, effective_date, amount) values (3,3,'2019-09-03',115500.0);
+insert into hr.employee_performance (employee_id, rating, evaluation_date) values (3,7.5,'2017-05-18');
+insert into hr.employee_performance (employee_id, rating, evaluation_date) values (3,2.5,'2019-06-13');
+insert into hr.employee_performance (employee_id, rating, evaluation_date) values (3,7.5,'2017-06-03');
+insert into hr.employee_performance (employee_id, rating, evaluation_date) values (3,7.0,'2019-09-17');
+insert into hr.employee_training (training_id, employee_id, training_name, start_date, end_date) values (1,3,'Training','2022-03-15','2022-03-16');
+insert into hr.employee_training (training_id, employee_id, training_name, start_date, end_date) values (2,3,'Training','2016-12-08','2016-12-10');
+insert into hr.employee_training (training_id, employee_id, training_name, start_date, end_date) values (3,3,'Training','2022-08-12','2022-08-14');
+insert into hr.employee_training (training_id, employee_id, training_name, start_date, end_date) values (4,3,'Training','2023-09-08','2023-09-09');
+insert into hr.employee_training (training_id, employee_id, training_name, start_date, end_date) values (5,3,'Training','2023-08-26','2023-08-28');
+insert into hr.employee_training (training_id, employee_id, training_name, start_date, end_date) values (6,3,'Training','2017-03-31','2017-04-03');
+insert into hr.employee_training (training_id, employee_id, training_name, start_date, end_date) values (7,3,'Training','2021-10-22','2021-10-25');
+insert into hr.employee_training (training_id, employee_id, training_name, start_date, end_date) values (8,3,'Training','2020-01-16','2020-01-16');
+insert into hr.employee_training (training_id, employee_id, training_name, start_date, end_date) values (9,3,'Training','2022-07-18','2022-07-18');
+insert into hr.employee_training (training_id, employee_id, training_name, start_date, end_date) values (10,3,'Training','2017-10-09','2017-10-09');
+insert into hr.employee_training (training_id, employee_id, training_name, start_date, end_date) values (11,3,'Training','2018-05-20','2018-05-23');
+insert into hr.employee_training (training_id, employee_id, training_name, start_date, end_date) values (12,3,'Training','2017-03-31','2017-04-03');
```

Comment:
I think you can use one insert with multiple values, no need to write many SQL queries here and below.